index.html

```diff
@@ -55,7 +55,7 @@ <h1>報告書を、<br class="br-sp">もっと使いやすく。</h1>
                   <label><input type="checkbox" class="ministry" value="14">金融庁</label><br>
                 </div>
                 <input type="search" id="searchWord" name="" autocomplete="off" placeholder="検索">
```

Comment:
検索欄のplaceholderとボタンの文言かぶるのが気になるので、こんなのはどうでしょうか

```suggestion
                 <input type="search" id="searchWord" name="" autocomplete="off" placeholder="調べたいキーワードを入力">
```

Reply:
そうしよう。こちらも今晩プルリク出します。